Tests/NetworkReachabilityManagerTests.swift

```diff
@@ -22,7 +22,7 @@
 //  THE SOFTWARE.
 //
 
-#if canImport(SytemConfiguration)
+#if canImport(SystemConfiguration)
```

Comment:
Thanks for the typo fix. I think also think `visionOS` needs to be added to the platforms for the WWAN tests at the bottom.